app/src/main/kotlin/com/exallium/todoapp/MainActivity.kt

```diff
@@ -59,4 +66,12 @@ class MainActivity : AppCompatActivity() {
             super.onBackPressed()
         }
     }
+
+    override fun onOptionsItemSelected(item: MenuItem?): Boolean {
```

Comment:
If it's not explicitly annotated as NotNull, android studio will assume it's nullable.  You can consult the docs to verify whether a null value would ever be passed =)